packages/hackmcx-api/src/utils/validation.js

```diff
@@ -0,0 +1,5 @@
+const whitespaceRegex = /^\s*$/.compile();
```

Comment:
```suggestion
const whitespaceRegex = new RegExp("/^\s*$/");
```
Apparently `RegExp.compile()` is deprecated and actually didn't work properly on my machine. If I'm not mistaken using the constructor instead gives us the same functionality and did end up working on my end at least.

Reply:
```
const whitespaceRegex = /^\s*$/;
```
Also works, `new RegExp()` might be redundant.